### DISCUSSION TITLE
加载祈愿数据报错

### DISCUSSION THREAD

**Discussion by wkby** [xunkong/KeqingNiuza#36] [Q&A] [answered]
我一共添加了三个账号，第一次添加的账号第二天更新祈愿数据报错
加载祈愿数据一直显示：未将对象弓|用设置到对象的实例。
删除此账号再加载还是一样报错
**这是log文件下的日志**
[keqing-211013.txt](https://github.com/Scighost/KeqingNiuza/files/7340219/keqing-211013.txt)
[keqing-211014.txt](https://github.com/Scighost/KeqingNiuza/files/7340220/keqing-211014.txt)
[DailyCheck.txt](https://github.com/Scighost/KeqingNiuza/files/7340221/DailyCheck.txt)
[keqing-211012.txt](https://github.com/Scighost/KeqingNiuza/files/7340222/keqing-211012.txt)
UserData文件下的东西放不进来

**这是报错详细信息截图**
![image](https://user-images.githubusercontent.com/92464810/137183645-6da624d8-0706-4193-b946-5a4045f91391.png)

**Comment by Scighost** [comment]
大概率是没有更新卡池信息出的错，运行 KeqingNiuza Launcher.exe

**Comment by wkby** [comment]
呜呜呜，正常了！太感谢了！就是米游社更新了，有了实时便签功能可以加上吗
![image](https://user-images.githubusercontent.com/92464810/137257824-8af6d929-2a04-4ab5-a785-e8df75bd260e.png)

**Comment by Lightczx** [comment]
#29 Here

**Comment by Zitingxuanzhu** [comment]
我的数据也报错了，导出了新池子数据后就报错了，一样的错误。。在记录json里把记录删除到只有一条也不行。。

我甚至尝试了重新下载，也不行。。

**Comment by Scighost** [comment]
> 我的数据也报错了，导出了新池子数据后就报错了，一样的错误。。在记录json里把记录删除到只有一条也不行。。
> 
> 我甚至尝试了重新下载，也不行。。

可能已经修复（2.1.5版本）

**Comment by Zitingxuanzhu** [comment]
> > 我的数据也报错了，导出了新池子数据后就报错了，一样的错误。。在记录json里把记录删除到只有一条也不行。。
> > 我甚至尝试了重新下载，也不行。。
> 
> 可能已经修复（2.1.5版本）

2.1.5.1版本“数据”没有问题，但“统计”还是报错。。
显示“未将对象引用设置到对象的实例”。。

`[Warning] [21-10-23 10:53:00.125 | KeqingNiuza 2.1.5.1 | 3BA58AB948D3357520E227CFE5CB417F] [SelectedUserData_Changed]
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 KeqingNiuza.ViewModel.WishSummaryViewModel..ctor(UserData userData) 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\ViewModel\WishSummaryViewModel.cs:行号 26
   在 KeqingNiuza.View.WishSummaryView.<UserControl_Loaded>b__9_0() 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\View\WishSummaryView.xaml.cs:行号 39
   在 System.Threading.Tasks.Task1.InnerInvoke()
   在 System.Threading.Tasks.Task.Execute()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 KeqingNiuza.View.WishSummaryView.<UserControl_Loaded>d__9.MoveNext() 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\View\WishSummaryView.xaml.cs:行号 39`

**Comment by Scighost** [comment, marked as answer]
> > > 我的数据也报错了，导出了新池子数据后就报错了，一样的错误。。在记录json里把记录删除到只有一条也不行。。
> > > 我甚至尝试了重新下载，也不行。。
> > 
> > 
> > 可能已经修复（2.1.5版本）
> 
> 2.1.5.1版本“数据”没有问题，但“统计”还是报错。。 显示“未将对象引用设置到对象的实例”。。
> 
> `[Warning] [21-10-23 10:53:00.125 | KeqingNiuza 2.1.5.1 | 3BA58AB948D3357520E227CFE5CB417F] [SelectedUserData_Changed] System.NullReferenceException: 未将对象引用设置到对象的实例。 在 KeqingNiuza.ViewModel.WishSummaryViewModel..ctor(UserData userData) 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\ViewModel\WishSummaryViewModel.cs:行号 26 在 KeqingNiuza.View.WishSummaryView.<UserControl_Loaded>b__9_0() 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\View\WishSummaryView.xaml.cs:行号 39 在 System.Threading.Tasks.Task1.InnerInvoke() 在 System.Threading.Tasks.Task.Execute() --- 引发异常的上一位置中堆栈跟踪的末尾 --- 在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() 在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 在 KeqingNiuza.View.WishSummaryView.<UserControl_Loaded>d__9.MoveNext() 位置 D:\a\KeqingNiuza\KeqingNiuza\src\KeqingNiuza\View\WishSummaryView.xaml.cs:行号 39`

今天遇到了一个和你一样的情况，猜测是因为设置文件有问题，他通过更改路径重新下载的方式解决了
